### PR TITLE
fix(client.scalar.com): localStorage isn’t enabled

### PR DESCRIPTION
**Pull request by amritk**
We are back!

### CODE REVIEW

**Review by marclave — APPROVED** [review]
we test on prod!

File: packages/api-client-app/electron.vite.config.ts [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { resolve } from 'path'
 export default defineConfig({
   main: {
     plugins: [externalizeDepsPlugin()],
+    define: {
```
Comment (amritk) [inline packages/api-client-app/electron.vite.config.ts:8]: @hanspagel this doesn't work, need a way to pass the env var in. Any ideas?

**Review by hanspagel — CHANGES_REQUESTED** [review]
Why does it have to be an env var, can’t we make this a regular setting? 🤔 The same way we pass a specification.

**Comment by marclave** [comment]
@hanspagel we can make it a different config later, its fine for now being a define : )

**Pull request merged**